google/cloud/storage/blob.py

```diff
@@ -361,6 +361,7 @@ def generate_signed_url(
         version=None,
         service_account_email=None,
         access_token=None,
+        use_virtual_hostname=False,
```

Comment:
Please use `virtual_host_style`

Reply:
Which spelling?  You typed `virtual_hosted_style` above. :)

Reply:
I picked this one, as it is shorter. LMK if I guessed wrong.

Reply:
Whoops :man_facepalming: use `virtual_hosted_style` so it's consistent across languages.

Reply:
Done: b75fd1d